### ISSUE TITLE
Could the package include pictures or would that make it too large...

### ISSUE THREAD

**Issue by brennanpincardiff** [CaRdiffR/tdor#3]
I've been exploring the magick package and it will allow the assembly of gifs. 
These could be made programmatically with selections based on country etc...

**Comment by brennanpincardiff** [comment]
This may not be the best way. 
It's feasible to pull the images from the website.
I'm exploring a script to do that...

**Comment by hturner** [comment]
Yes this would be preferable. For now the package only contains the spreadsheet data, which includes names of jpg files - maybe that is useful in your script?

**Comment by sastoudt** [comment]
Is it possible to add the Bitbucket URLs for accessing the photos to the data in the R package? If so, I think I can add photos to the map in the Shiny Dashboard like the website currently does, [without having the photos locally](https://stackoverflow.com/questions/36433899/image-in-r-leaflet-marker-popups).

**Comment by hturner** [comment]
Hi folks, I've discovered you can link to the photos as follows:

https://bytebucket.org/annajayne/tdor_data/raw/default/Data/TDoR%202007/photos/2007_07_20_Victoria-Arellano.jpg

where `2007_07_20_Victoria-Arellano.jpg` is value in the `Photo` variable of the `tdor` data and `TDoR%202007` is formed from the relevant TDoR period, as in the `TDoR` variable. We should update the `Photo` variable in the dataset to be more useful, but in the meantime perhaps you can create the links yourself for testing things out? (Or if someone wants to update the data, feel free!)

**Comment by annajayne** [comment]
Just as a heads up I'm likely to re-organise the BitBucket data once I've finished entering reports back to 1999 - the idea is to move it to a yearly basis rather than on the basis of TDoR period (which changes as you go further back, then becomes meaningless before 1999).

As such the permanent links to individual files will change.
